.github/workflows/pr-pull.yaml

```
@@ -1,4 +1,4 @@
-name: Pull Request
+name: PR
```

```suggestion
name: Pull
```

I've looked at the other files and also logically went over this, and the correct label is "Pull", because the workflow actually "pulls" the pull-_request_ and compiles it. "PR" would be semantically wrong, and would also go against the naming scheme of the other workflows.

---

Applied change.